modules/ossm-creating-a-default-revision-tag-and-relabeling-the-namespaces.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-creating-a-default-revision-tag-and-relabeling-the-namespaces_{context}"]
+= Creating the default revision tag and relabeling the namespaces
+
+This procedure explains how to create the default revision tag and relabel the namespaces after you have completed the {SMProduct} 2 to {SMProduct} 3 cluster-wide migration process using istio injection label.
+
+The `bookinfo` example application is being used for demonstration purposes with a minimal example for the `Istio` resource. For more information on configuration differences between the {SMProduct} 2 `ServiceMeshControlPlane` resource and the {SMProduct} 3 `Istio` resource, see "Configuration fields mapping between Service Mesh 2 and Service Mesh 3".
```

Comment:
```suggestion
The `bookinfo` example application is being used for demonstration purposes.
```

Reply:
We are not touching Istio resource here.

Reply:
Done